blocks/tabs/tabs.css

```diff
@@ -10,7 +10,7 @@ main .tabs {
 
 main div.tab-item:not(.active) {
     animation: fade-out 0ms linear forwards;
-
+    display: none;
```

Comment:
I am wondering why it was not present earlier and how tab switching is working fine in other devices

Reply:
Perhaps the `animation` property above is making it behave differently but not sure. I observed that the height is 0 for inactive tabs in chrome and hence `display: none` is not needed.